src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/MainLayout.Auth.razor

```diff
@@ -11,8 +11,8 @@
             <a href="https://docs.microsoft.com/aspnet/" target="_blank">About</a>
         </div>
 
-        <div class="content px-4">
+        <article class="content px-4">
             @Body
-        </div>
+        </article>
```

Comment:
Changing `div` to `article` here doesn't actually have any functional impact that I was able to confirm practically using a screen reader. However, the advice to use semantic landmarks for things like "the main content" is pretty universal so I don't see any reason for us not to do it - it causes no drawbacks that I can identify.

Reply:
Same with changing `<div class="main">` around it to just `<main>`.